packages/nuxt/.storybook/addons.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Do we need Storybook in Nuxt package?

Reply:
I think it can be helpful to see all components separately in nuxt env.

Reply:
Ok, as we discussed I'll remove Storybook from this package.